### PR TITLE
Image adjustment and CSS fix

### PR DESCRIPTION
**Pull request by rqk2k**
- CSS border on each number.
- Space between numbers
- Adjustment of image size

### CODE REVIEW

File: src/components/countdown/cop-15.vue [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
             </a>
         </div>
-        <div>
+        <div class="alignment">
             <a :href="url">
-                <span v-if="time.days > 1">{{ time.days }} DAYS<br />TO COP 15</span>
-                <span v-else-if="time.days == 1">{{ time.days }} DAY<br />&nbsp;TO COP 15</span>
+                <span v-if="time.days > 1"><span class="countdown-box">{{ time.days.split('')[0] }}</span>&nbsp;<span class="countdown-box">{{ time.days.split('')[1] }}</span> DAYS<br />TO COP 15</span>
+                <span v-else-if="time.days == 1"><span class="countdown-box">{{ time.days.split('')[0] }}</span>&nbsp;<span class="countdown-box">{{ time.days.split('')[1] }}</span> DAY<br />&nbsp;TO COP 15</span>
```
Comment (sbilodeau) [inline src/components/countdown/cop-15.vue]: simplify this :)

**Review by sbilodeau — CHANGES_REQUESTED** [review]

File: src/components/countdown/cop-15.vue (hunk 1 above)
Comment (rqk2k) [reply src/components/countdown/cop-15.vue]:
```          
                <span v-if="time.days > 1">
                    <span v-for="char in time.days">
                        <span class="countdown-box">{{ char }}</span>&nbsp;
                    </span>
                    DAYS<br />TO COP 15
                </span>
                <span v-else-if="time.days == 1">
                    <span v-for="char in time.days">
                        <span class="countdown-box">{{ char }}</span>&nbsp;
                    </span>
                    DAY<br />TO COP 15
                </span>
                <span v-else>COP 15</span>
```

Will this work?

Comment (blaisefonseca) [reply src/components/countdown/cop-15.vue]: maybe just this
```
<span>
      <span v-for="char in time.days">
          <span class="countdown-box">{{ char }}</span>&nbsp;
      </span>
      DAY<span  v-else-if="time.days > 1">S</span>
      <br />TO COP 15
 </span>
 <span v-else>COP 15</span>
```

Comment (rqk2k) [reply src/components/countdown/cop-15.vue]: Thank you @blaisefonseca, please find the revised version below.

```
                <span v-if="time.days >= 1">
                    <span v-for="char in time.days"><span class="countdown-box">{{ char }}</span>&nbsp;</span>DAY<span v-if="time.days > 1">S</span><br />TO COP 15
                </span>
                <span v-else>COP 15</span>
```

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**